src/formatter/helpers/event_data_collector.ts

```diff
@@ -20,13 +21,18 @@ export interface ITestCaseAttempt {
   testCase: messages.ITestCase
 }
 
+export interface IEventBroadcaster {
+  on(name: string, handler: Function): void
+  emit(name: string, envelope: IEnvelope): void
+}
```

Comment:
Don't think this is needed. Have just used `EventEmitter` thus far. Can we stick with that for now and if we want to change that do so in a separate PR?